modules/test/playwright/tests/site-admin-web/pages/MembershipsPage.ts

```diff
@@ -17,6 +17,19 @@ export class MembershipsPage {
 		this.productMenuPage = new ProductMenuPage(page);
 	}
 
+	async addAllUsersToMembership() {
```

Comment:
Can this be renamed to `addAllUsersToSite` for clarity? Or maybe `assignAllUsersSiteMembership`?